data_analysis/data_analysis.py

```diff
@@ -0,0 +1,231 @@
+"""
+This python files performs data analysis on the canproco dataset.
```

Comment:
I suggest to be more verbose-- if someone external contributes to this project, it is unclear what "analysis" means. Eg you could say something like: check the number of subjects per time points and per MRI contrast (PSIR, STIR, etc.).